airbyte-local.sh

```diff
@@ -13,64 +13,6 @@ if ((${missing_require:-0})); then
     exit 1
 fi
 
-pagerduty_src_catalog=$(cat << EOF
-{
-  "streams": [
-    {
-      "stream": {
-        "name": "incident_log_entries"
-      },
-      "sync_mode": "incremental",
-      "destination_sync_mode": "append"
```

Comment:
looking into a discrepancy between where these properties live here vs [here](https://github.com/faros-ai/airbyte-connectors-configs/blob/main/catalogs/servicenow.json#L17-L20)

Reply:
Added a jq line to transform to how we expect it.